docs/details/advisors/checks/mongodb-scram-sha-256.md

```diff
@@ -0,0 +1,54 @@
+# MongoDB not using the default SHA-256 hashing function 
+
+## Description
+This check warns if a modern authentication method is not used. 
+The goal is to follow updated and optimal security processes. 
+
+For Production systems: ensure that auth and authentication methods are in place.
+
+MongoDB enables journaling by default and  "authenticationMechanisms" are set to use "SCRAM-SHA-256" by default in  MongoDB  versions (4.0 +).
+MongoDB v3.0changed the default auth mechanism from MONGODB-CR to SCRAM-SHA-1.
+[Authentication Mechanisms in the MongoDB documentation](https://docs.mongodb.com/drivers/go/current/fundamentals/auth/).
```

Comment:
This links leads to golang driver documentation. Probably better to use this link instead: https://www.mongodb.com/docs/manual/core/authentication/#authentication